db/setup/schema.sql

```diff
@@ -0,0 +1,10 @@
+CREATE DATABASE ft_transcendence;
+
+\c ft_transcendence;
+
+CREATE TABLE IF NOT EXISTS users(
+    UUID        UUID DEFAULT gen_random_uuid()  PRIMARY KEY,
+    USERNAME    VARCHAR(20)                     NOT NULL,
+    EMAIL       VARCHAR(50)                     NOT NULL,
```

Comment:
suggestion: We could make these fields unique.

Reply:
Nice idea, I'll do it!

Reply:
done!